.claude/skills/screenize-explorer/SKILL.md

```
@@ -0,0 +1,168 @@
+When invoked, follow these steps. All output must be in English.
+
+## Step 1: Find Project
+
+If the user provides a specific `.screenize` path as an argument, use it directly.
+
+Otherwise, search for projects in these locations (in order):
+1. `./projects/` relative to the current working directory
+2. `~/Movies/Screenize/`
+
+Use Glob to find `.screenize` packages:
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)